src/model/src/InertialParametersHelpers.cpp

```diff
@@ -105,6 +109,8 @@ void buildMesh(const aiScene* scene, const aiNode* node, const double scale, std
 
 bool BBFromExternalShape(ExternalMesh* extMesh, Box& box)
 {
+    
+    
```

Comment:
Not a big problem in this case, but in general it is a good idea to avoid to include unrelated changes in a PR, to simplify the reviewers' job.